src/main/psxsdk/libsnd/vmanager.c

```diff
@@ -11,7 +11,47 @@ INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/vmanager", SpuVmDoAllocate);
 
 INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/vmanager", vmNoiseOn);
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/vmanager", vmNoiseOn2);
+void vmNoiseOn2(u8 arg0, u16 arg1, u16 arg2, u16 arg3, u16 arg4) {
```

Comment:
I simplified the function a bit: https://decomp.me/scratch/8n4mp

Also I think `arg0` can become `voice`? and the existing `voice` can become `vc`, `i`, `index` or something similar?